### PR TITLE
[Routing] Add FQCN and FQCN::method aliases when applicable

### LINKED ISSUE #18437: [Routing] Add FQCN and FQCN::method aliases when applicable
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#50084
| PR author(s) | @fancyweb
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/18437

### CODE REVIEW

File: routing.rst [hunk 1]
```diff
@@ -2236,6 +2236,34 @@ For that reason each route has an internal name that must be unique in the
...
+
+        // src/Controller/MainController.php
+        namespace App\Controller;
+
+        use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+        use Symfony\Component\Routing\Annotation\Route;
+
+        class MainController extends AbstractController
+        {
+            #[Route('/', name: 'homepage', stateless: true)]
```
Comment (OskarStark) [inline routing.rst]: the stateless is not important here, isn't it?

File: routing.rst [hunk 2]
```diff
@@ -2236,6 +2236,34 @@ For that reason each route has an internal name that must be unique in the
...
+one route exactly. Symfony also automatically adds a alias for every method
+that defines only one route. This means that for the following class::
+
+        // src/Controller/MainController.php
+        namespace App\Controller;
+
+        use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+        use Symfony\Component\Routing\Annotation\Route;
+
+        class MainController extends AbstractController
```
Comment (OskarStark) [inline routing.rst]:
```suggestion
        final class MainController extends AbstractController
```

File: routing.rst (hunk 1 above)
Comment (alexandre-daubois) [reply routing.rst]: That's right, updated :+1:

**Comment by javiereguiluz** [comment]
Alex, thanks for contributing this feature!

**Pull request merged**